main.nf

```diff
@@ -201,29 +193,30 @@ process mask_beds {
     module "$params.modules.bedtools"
 
     """
-    BNAME=\$( echo $bedfile | cut -d. -f1 )
-    MASK_FILE=\${BNAME}_masked.bed
-    cat $bedfile \
-        | bedtools intersect -v -a stdin -b $mask1 -f 0.25 \
-        | bedtools intersect -v -a stdin -b $mask2 -f 0.25 > \$MASK_FILE
+    BNAME=\$( echo $svfile | cut -d. -f1 )
```

Comment:
Or `\${svfile%%.*}` if `svfile` is a shell variable, which I think it ought to be.